mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionDaoImpl.java

```diff
@@ -205,6 +206,19 @@ public Future<JobExecution> updateBlocking(String jobExecutionId, JobExecutionMu
     return promise.future();
   }
 
+  @Override
+  public Future<Boolean> deleteJobExecutionsByIds(List<String> ids, String tenantId) {
+    Promise<RowSet<Row>> promise = Promise.promise();
+    try {
+      String query = format(DELETE_BY_IDS_SQL, convertToPsqlStandard(tenantId), TABLE_NAME, ids.stream().collect(Collectors.joining("','")));
+      pgClientFactory.createInstance(tenantId).execute(query, promise);
+    } catch (Exception e) {
+      LOGGER.error("Error deleting jobExecution by ids {} : Exception : ", ids, e);
```

Comment:
Hi @SerhiiNosko @SatishkumarPurswani , we don't have "entity" here, so we can't use it.